ui/app/templates/components/clients/attribution.hbs

```diff
@@ -40,7 +43,7 @@
         <p class="data-details">{{this.topClientCounts.label}}</p>
       </div>
 
-      <div class="data-details-bottom" data-test-top-counts>
+      <div class="data-details-bottom" data-test-attribution-clients>
```

Comment:
this seems to be missing from the original PR?  
![image](https://user-images.githubusercontent.com/6618863/171969675-9a43a885-8130-44e6-85c1-5dc3ae427835.png)

Reply:
It's there, just not in the dif. The difs vary slightly because the original PR is compared against main, and this backport is compared to the `release/1.11.x` branch